### PR TITLE
Allow disabling of null handling

### LINKED ISSUE #82: Prohibit empty values and nulls with a special option
Right now TOML does not allow null or empty values (nullable types). But ktoml supports it, sowe should make it configurable (enabled by the default)

### PR DESCRIPTION
**Pull request by bishiboosh**
Fixes #82

### CODE REVIEW

**Comment by akuleshov7** [comment]
Wow! Thank you so much for these two pull requests! This is awesome

File: ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/parsers/ValueParserTest.kt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.akuleshov7.ktoml.parsers
 
+import com.akuleshov7.ktoml.TomlConfig
```
Comment (akuleshov7) [inline ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/parsers/ValueParserTest.kt:3]: Could you please also add a "decoder" test in the decoder test directory?
And fix existing, if it is possible for you 🙏

File: README.md [hunk 2]
```diff
@@ -192,13 +192,15 @@ special configuration class that can be passed to the decoder method:
...
 Toml(
-    ktomlConf = KtomlConf(
+    config = TomlConfig(
         // allow/prohibit unknown names during the deserialization, default false
         ignoreUnknownNames = false,
         // allow/prohibit empty values like "a = # comment", default true
-        emptyValuesAllowed = true,
+        allowEmptyValues = true,
+        // allow/prohibit null values like "a = null", default true
+        allowNullValues = false,
```
Comment (akuleshov7) [inline README.md:201]: Better to put here the same default value (as we have in `TomlConfig`)

**Review by akuleshov7 — APPROVED** [review]

**Comment by akuleshov7** [comment]
Thank you so much for your contribution! @bishiboosh don't you mind if I will invite you to maintainers of this repo?

**Comment by bishiboosh** [comment]
No, please do ! I don't know if I'll be able to contribute that much but I'd be happy to help !

**Pull request merged**